string model
uint8 temperature
uint16 Spin_speed
float32 Kilogram
bool NormalTM2L7
bool DailyTM2L7
bool SuperQuickTM2L7
bool Economy
bool Prewash
bool Stain
bool ExtraRinse
bool Soft
bool Night_Cycle
bool Rinse_Hold
bool Steam_Anticrease
bool Low_Steam
bool Medium_Steam
bool Steam_Max
uint32 User_select_Rinse_Nums
uint16 User_Select_Delay_Time
uint8 User_Select_Program_ID
bool Prewashing
bool Washing
bool Rinsing
bool Spinning
bool Drying
bool Delay
bool Steaming
bool Anticrease
uint8 program
uint8 option
